src/utils/requests/UserRequests.js

```diff
@@ -1,23 +1,59 @@
+const { URL } = require("../../config")
+
 /**
  * @function logUser
  * @description Function used to wrap the login request while simply requiring credentials as params
  * @param {String} credentials.email
  * @param {String} credentials.password // no need to encrypt yet
  */
 export const login = async (credentials) => {
-    // should do something with credentials...
-    // hint: use GET
+    try { 
+        let response = await fetch(URL + "/login", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(credentials),
+        });
+
+        return await checkStatus(response);
+
+    } catch(error){
+        return error;
```

Comment:
so we still want the `login` function to throw if an error occurs during the HTTP request (perhaps we can add an error pop up if it fails).

```suggestion
        throw error;
```